taskvine/src/manager/vine_file_replica.c

```diff
@@ -17,10 +18,13 @@ struct vine_file_replica *vine_file_replica_create(
 	r->transfer_time = 0;
 	r->last_failure_time = 0;
 	r->state = VINE_FILE_REPLICA_STATE_PENDING;
+
+	vine_counters.replica.create++;
 	return r;
 }
 
 void vine_file_replica_delete(struct vine_file_replica *r)
 {
 	free(r);
+	vine_counters.replica.delete ++;
```

Comment:
extra space (here and in other places below).

Reply:
Odd, the clang formatter is insisting upon that.
Perhaps it is considering `delete` as a C++ keyword?
(and perhaps I should change the symbol name.)